extensions/amp-form/0.1/amp-form.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software'
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-form .amp-hidden {
+    display: none;
+}
+
+amp-form amp-field {
+    display: block;
```

Comment:
This might require UX input, but since AMP is focused on mobile currently, I feel defaulting to a mobile friendly UX for inputs and labels would be a good thing. This can be overridden by the custom styles publishers provide, but I feel having a really nice default would be great.